source/gui/guiHelper.py

```diff
@@ -124,12 +124,30 @@ def addButton(self, *args, **kwargs):
 		return wxButton
 
 
-def associateElements(firstElement: wx.Control, secondElement: wx.Control) -> wx.BoxSizer:
+# vertical controls where the label should go above visually, and the control should go below
+_VerticalCtrlT = TypeVar("_VerticalCtrlT", wx.ListCtrl, wx.ListBox, wx.TreeCtrl)
```

Comment:
Can we rename this to use the full word?
```suggestion
_VerticalControlT = TypeVar("_VerticalCtrlT", wx.ListCtrl, wx.ListBox, wx.TreeCtrl)
```

Reply:
I'd prefer to use wx syntax here for consistency